test/test_nn.py

```diff
@@ -4183,11 +4189,13 @@ def eval_constructor(*args, **kwargs):
 
 for test_params in criterion_tests + new_criterion_tests:
     name = test_params.pop('module_name')
-    test_params['constructor'] = getattr(nn, name)
+    ctor = getattr(nn, name)
+    test_params['constructor'] = ctor
```

Comment:
why split this into two lines?

Reply:
`ctor` is reused below to make the constructors of the no_size_average and no_reduce tests